apps/lib/signer/signer.go

```diff
@@ -185,6 +200,92 @@ func (s *StarkSigner) GetSignatureType() SignatureType {
 	return StarkSignatureType
 }
 
+type StorkAuthSigner interface {
+	SignAuth(publishTimestamp int64) (string, string, string, error)
+	GetAuthHeaders() (http.Header, error)
+}
+
+type EvmAuthSigner struct {
+	evmSigner *EvmSigner
+}
+
+func NewEvmAuthSigner(privateKeyStr EvmPrivateKey, logger zerolog.Logger) (*EvmAuthSigner, error) {
+	evmSigner, err := NewEvmSigner(privateKeyStr, logger)
+	if err != nil {
+		return nil, err
+	}
+	return &EvmAuthSigner{evmSigner: evmSigner}, nil
+}
+
+func (s *EvmAuthSigner) SignAuth(publishTimestamp int64) (string, string, string, error) {
+	timestampedSignature, _, err := s.evmSigner.SignPublisherPrice(publishTimestamp, StorkAuthAssetId, StorkMagicNumber)
```

Comment:
calls the existing SignPublisherPrice function but with a fixed asset id and quantizedValue